libs/partners/ai21/tests/integration_tests/test_chat_models.py

```diff
@@ -0,0 +1,47 @@
+"""Test ChatAI21 chat model."""
+import pytest
+from langchain_core.messages import HumanMessage
+from langchain_core.outputs import ChatGeneration
+
+from langchain_ai21.chat_models import ChatAI21
+
+
+@pytest.mark.requires("ai21")
```

Comment:
can remove these requires statements - will always be there in partner package!

Reply:
Done